backend/internal/test/integration/serverexperimentation/testrunner.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.16.0
+
+COPY . /code
+
+RUN cd /code/module/chaos/serverexperimentation && find . -type f -name "*_test.go" ! -name "*_integration_test.go" -exec rm {} \;
```

Comment:
is this necessary when using the build tag?

Reply:
seems to work without them 🤷  removed